addons/l10n_ar/data/l10n_latam.document.type.csv

```diff
@@ -71,18 +71,18 @@ dc_ac_inc_cf,690,94,AJUSTES CONTABLES QUE INCREMENTAN EL CREDITO FISCAL,,,,,base
 dc_ac_dis_cf,700,95,AJUSTES CONTABLES QUE DISMINUYEN EL CREDITO FISCAL,,,,,base.ar,,False
 dc_f1116b,710,96,FORMULARIO 1116 B,,,,,base.ar,,False
 dc_f1116c,720,97,FORMULARIO 1116 C,,,,,base.ar,,False
-dc_oc_ncrg3419,730,99,OTROS COMP  QUE NO CUMPLEN CON LA R G  3419 Y SUS MODIF,,,invoice,OC-X,base.ar,not_zero,True
+dc_oc_ncrg3419,730,99,OTROS COMPROBANTES QUE NO CUMPLEN O ESTAN EXCEPTUADOS DE LA R.G. 1415 Y SUS MODIF ,,,invoice,OC-X,base.ar,not_zero,True
```

Comment:
@Bruno-Zanotti y would remove the spaces on `MODIF ` I know that afip table has this space, but better to remove it

Reply:
:+1: